feature-libs/order/assets/translations/en/index.ts

```diff
@@ -4,8 +4,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { order } from './order.i18n';
+import { order, orderEnhancedUI } from './order.i18n';
 
 export const en = {
   order,
+  orderEnhancedUI,
```

Comment:
[MAJOR]
Consistent qualifier suffix for the new classes/translations/services/etc. - to be decided.
Ideas:
OrderHistory**Customer360**Module
OrderHistory**V2**Module
OrderHistory**Extended**Module
OrderHistory**EnhancedUi**Module
OrderHistory**ExtendedView**Module

Currently we're the most fond of **Extended**. Let's wait for answer from stakeholders about the idea of **Customer360**.

Reply:
new proposition,using an unique code name:
OrderHistory*EnhancedMyAccount*Module

Reply:
will change all names with prefix 'MyAccountV2' as discussed